### PR TITLE
Update links in A9 Tutorial description

### LINKED ISSUE #234: A9 - Wrong url, to be removed
Hello,

The 6th link [bithound](http://www.bithound.io/) should be removed from the [page ](https://nodegoat.herokuapp.com/tutorial/a9)as it's not a Nodejs service anymore but a casino website. 

Cheers,
Genia.

### PR DESCRIPTION
**Pull request by rcowsill**
Replaced stale links with appropriate alternatives and added dependabot links.

Fixes #234

### CODE REVIEW

File: app/views/tutorial/a9.html [hunk 1]
```diff
@@ -22,28 +22,44 @@ <h3 class="panel-title">Description</h3>
...
+                    <ol>
                         <li>
-                            <a href="#npm outdated">npm outdated</a> as well as 'yarn outdated' are both command line ways to show possibly out of date dependencies
+                            <a href="https://docs.npmjs.com/cli/v6/commands/npm-outdated">npm outdated</a> and <a href="https://classic.yarnpkg.com/en/docs/cli/outdated">yarn outdated</a> are both command line ways to show possibly out of date dependencies
+                        </li>
+                        <li>
+                            <a href="https://docs.github.com/en/code-security/supply-chain-security/about-dependabot-version-updates">Dependabot version updates</a> can automatically make GitHub pull requests to update outdated dependencies
+                        </li>
+                        <li>
+                            <a href="https://david-dm.org/">David DM</a> gets you an overview of your project dependencies, the version you use and the latest available, so you can quickly see what's drifting
```
Comment (lirantal) [inline app/views/tutorial/a9.html:57]: david-dm is an unmaintained project and we can remove it

Comment (rcowsill) [reply app/views/tutorial/a9.html:57]: Is it unmaintained? I didn't see anything to suggest that at either https://david-dm.org or https://github.com/alanshaw/david-www.

Comment (lirantal) [reply app/views/tutorial/a9.html:57]: Ahh, maybe this went back into active development. I remember at point in time seeing how this was largely unmaintained.

Comment (lirantal) [reply app/views/tutorial/a9.html:57]: Seems like it has resumed activity
image

**Pull request merged**